### ISSUE TITLE
FileName --> Assert error raised by openCV on open CvCapture

### ISSUE THREAD

**Issue by HuguesDug** [Laex/Delphi-OpenCV#118]
Hello,

Following your comment on "go for OpenCV 3.0", I tried to install openCV 3.4.5 on my computer, x64 version only, as binaries are not available in x84.

In updated the opencv.inc to reflect this change of version.

I recompiled some code using a FileName:TFileName to open capture with  cvCreateFileCapture(PAnsiChar(@(AnsiString(FileName)[1])));
(as you do in ocv.comp.source)

I have a nice assert Error : 
File minkernel\crts\ucrt\appcrt\convert\isctype.cpp
Line: 36
Expression c>= -1 && c <=255

Any idea ?

**Comment by Laex** [comment]
No no no
Do not try to run version 3.0.
Everything should be done differently there ...
What is in the repository is an early attempt at translation. It does not work.

**Comment by blaisexen** [comment]
Actually OpenCv_345 x86 and x64  works on Delphi-OpenCV,
I tried some or few functions like,....
cvShowImage('x', cvLoadImageM('test.jpg', CV_LOAD_IMAGE_COLOR ));

So, I got OpenCv_World345.dLL Only and no other DLL(s), and myExE.

if ONLY cvCreateCameraCapture(on 345) still works then I'll be using OpenCv_345 for face detection with Camera.

Credits to Master Laex, without you there is no OpenCv for Delphi!

I also tried running from c++Builder, again cvCreateCameraCapture does not work with OpenCv_345


p.S
cvCreateFileCapture does not work also
